spec/unit/application/lookup_spec.rb

```diff
@@ -145,6 +181,34 @@
       lookup.options[:explain] = true
       lookup.command_line.stubs(:args).returns(['a'])
       expect { lookup.run_command }.to output(<<-EXPLANATION).to_stdout
+Searching for 'lookup_options'
+  Merge strategy hash
+    Data Binding "hiera"
+      No such key: "lookup_options"
+    Data Provider "Hiera Data Provider, version 4"
+      ConfigurationPath "#{environmentpath}/production/hiera.yaml"
+      Data Provider "common"
+        Path "#{environmentpath}/production/data/common.yaml"
+          Original path: common
+          No such key: "lookup_options"
+Merge strategy first
+  Data Binding "hiera"
+    No such key: "a"
+  Data Provider "Hiera Data Provider, version 4"
+    ConfigurationPath "#{environmentpath}/production/hiera.yaml"
+    Data Provider "common"
+      Path "#{environmentpath}/production/data/common.yaml"
+        Original path: common
+        Found key: "a" value: "This is A"
+  Merged result: "This is A"
```

Comment:
Do we have to say "Merged result". Isn't "Result" clearer? (Otherwise it leaves the question "as opposed to what, the 'result' - are there different results?"